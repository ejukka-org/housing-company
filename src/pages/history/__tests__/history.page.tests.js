import React from 'react';
import HistoryPage from '../history.page';
import renderer from 'react-test-renderer';

test('Renders History page', () => {
    const component = renderer.create(
        <HistoryPage />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
