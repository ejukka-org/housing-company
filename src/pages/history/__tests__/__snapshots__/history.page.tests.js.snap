// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders About page 1`] = `
<div
  className="container"
>
  <div
    className="container"
  >
    <video
      controls={true}
      height="240"
      hidden={false}
      poster="talo1.png"
      width="320"
    >
      <source
        src="https://media.w3.org/2010/05/sintel/trailer_hd.mp4"
        type="video/mp4"
      />
      Your browser does not support the video tag.
    </video>
  </div>
</div>
`;

exports[`Renders History page 1`] = `
<div
  className="container"
>
  <div
    className="container"
  >
    <video
      controls={true}
      height="240"
      hidden={false}
      poster="talo1.png"
      width="320"
    >
      <source
        src="https://media.w3.org/2010/05/sintel/trailer_hd.mp4"
        type="video/mp4"
      />
      Your browser does not support the video tag.
    </video>
  </div>
</div>
`;
